src/Controller/admin.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Controller;
```

Comment:
zły namespace! App\Controller

Reply:
To zostało poprawione w następnym PR, ale postaram się to poprawić.